inputs/packagelock.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation and others.
+// SPDX-License-Identifier: Apache-2.0
+
+import MetadataSource from './base';
+import { Package } from '../structure';
+
+export default class PackageLockSource implements MetadataSource {
+  private packageMap: Map<string, Package>;
+
+  constructor(json: string) {
+    const content = JSON.parse(json);
+    const dependencies = Object.keys(content.dependencies).map(x => {
```

Comment:
Doesn't this only capture direct dependencies, and not any transitive dependencies? Package locks are recursive.

Reply:
yep - I overlooked that. good call

Reply:
@jpeddicord  updated :)